_site/index.html

```diff
@@ -85,39 +84,39 @@ <h2>A multi-cluster Control Plane for defining, configuring and deploying Ingres
   <div class="container" data-aos="fade-up">
 
     <div class="section-title">
-      <h2>Features</h2>
-      <p>Explore the wide range of features that Kuadrant offers for both <b>Gateway Administrators</b> and <b>Application Developers</b></p>
+      <h2>Feature Overview</h2>
+      <p>Explore the wide range of features that Kuadrant offers to enable and empower <b>cluster operators</b> and <b>application developers</b> to work together to securely connect users with their services. Our goal is to empower cluster administrators to deploy and manage gateways at scale to provide application connectivity and give them the confidence to allow application developers to self service knowing that endpoints exposed via these gateways are compliant, secure and resilient. All of this is underpinned by powerful new ingress standard <a href="https://gateway-api.sigs.k8s.io/">Gateway API.</a> and the concepts behind <a href="https://gateway-api.sigs.k8s.io/references/policy-attachment/">policy attachment.</a>
     </div>
 
     <div class="row">
       <div class="col-xl-3 col-md-6 d-flex align-items-stretch" data-aos="zoom-in" data-aos-delay="100">
         <div class="icon-box">
           <div class="icon"><i class="bx bx-globe"></i></div>
-          <h4><a href="">Multi-Cluster Ingress with DNS</a></h4>
-          <p>Establish seamless connectivity and advanced DNS management for workloads in multi-cluster deployment models with Kuadrant.</p>
+          <h4><a href="">Multi-Cluster Gateways connectivity and Load Balancing</a></h4>
```

Comment:
maybe: "Multi-Cluster Gateways connectivity and Load Balancing" -> "Multi-Cluster Gateways: Connectivity and Load Balancing"?